indico/modules/events/views.py

```diff
@@ -120,9 +120,43 @@ def _getHeader(self):
     def getJSFiles(self):
         return WPDecorated.getJSFiles(self) + self._asset_env['modules_event_display_js'].urls()
 
+    def _getJsonLDData(self):
+        lddata = {
+            "@context": "http://schema.org",
+            "@type": "Event",
+            "url": self.event.external_url,
+            "name": self.event.title,
+            "startDate": self.event.start_dt.isoformat(),
+            "endDate": self.event.end_dt.isoformat(),
+            "location": {
+                "@type": "Place",
+                "name": self.event.venue_name or 'No location set',
+                "address": self.event.address or 'No address set'
+                },
+            "description": strip_tags(self.event.description),
+            "performers": self._getJsonLDPerformers(self.event.person_links)
+        }
+        if self.event.has_logo:
+            lddata["image"] = url_for('event_images.logo_display', self.event, slug=self.event.logo_metadata['hash'],
```

Comment:
Maybe you could add a `external_logo_url` property (right below the `logo_url`) property to `Event` and use that here?